one.js

```diff
@@ -25,6 +55,18 @@ const lightTvs = inventory.filter((ambiTv) => {
 
 console.log(lightTvs);
 
+//1c. in een functie voor bonusopdracht//
+
+function lightTvsF(array) {
+    return array.filter((ambiLightTV) => {
+        if (ambiLightTV.options.ambiLight === true) {
+            return true
+        }
+    })
```

Comment:
Deze zou je nog wat korter kunnen schrijven
```suggestion
    return array.filter(ambiLightTV => ambiLightTV.options.ambiLight);
```